libraries/ms-helper/src/can_uart.c

```diff
@@ -0,0 +1,106 @@
+#include "can_uart.h"
+#include <string.h>
+#include "log.h"
+
+// CTX in ASCII
+#define CAN_UART_TX_MARKER 0x585443
```

Comment:
Ah I see, so `CAN_UART_TX_MARKER` and `CAN_UART_RX_MARKER` are directives for the receiver that tell it how to handle the message?

Reply:
Yeah, pretty much. That also allows other data to be carried such as GPS.